File: Bio Pro/BioProFrontend/src/app/shared/EmployeeHoursReport.ts
// shared/EmployeeMonthlyHours.ts
export interface EmployeeWeeklyHours {
  employeeId: number;
  firstName: string;
  lastName: string;
  week: number;
  totalHours: number;
}

export interface EmployeeHoursReport {
  firstName: string;
  lastName: string;
  year: number;
  month: number;
  week: number;
  totalHours: number;
}

File: Bio Pro/BioProSystem/BioProSystem/ViewModels/EmployeeHoursReport.cs
using BioProSystem.Models;

namespace BioProSystem.ViewModels
{
    public class EmployeeHoursReport
    {
        public Employee Employee { get; set; }
        public int Year { get; set; }
        public int Month { get; set; }
        public int Week {  get; set; }
        public decimal TotalHours { get; set; }
    }
}
